tests/unit/utils/test_terraform_utils.py

```diff
@@ -111,11 +112,14 @@ def test_enable_key_function_handles_components_without_flavors(
     """
     comp_flavor = "s3"
     comp_type = "artifact_store"
+    comp_provider = "aws"
     c = Component(
         name=dummy_name,
         component_flavor=comp_flavor,
         component_type=comp_type,
-        provider=random.choice(list(ProviderEnum)).value,
+        # provider=random.choice(list(ProviderEnum)).value,
+        # Not sure why the above line was used when only "aws" is valid here
```

Comment:
I think that's because component providers might not have been validated previously.

Reply:
maybe delete the commented-out lines then?